src/current/_includes/releases/v23.1/v23.1.11.md

```diff
@@ -1,9 +1,9 @@
 ## v23.1.11
 
 {{ site.data.alerts.callout_info }}
-v23.1.11 is currently available only on select CockroachDB {{ site.data.products.dedicated }} clusters. If you wish to upgrade your {{ site.data.products.dedicated }} cluster to v23.1.11, please submit a [support request](https://support.cockroachlabs.com/hc/en-us/requests/new).
+v23.1.11 is currently available only for CockroachDB {{ site.data.products.serverless }} customers and select {{ site.data.products.dedicated }} clusters. If you wish to upgrade your {{ site.data.products.dedicated }} cluster to v23.1.11, please [submit a support request](https://support.cockroachlabs.com/hc/en-us/requests/new?).
```

Comment:
```suggestion
v23.1.11 is currently available for CockroachDB {{ site.data.products.serverless }} customers and select {{ site.data.products.dedicated }} clusters. If you wish to upgrade your {{ site.data.products.dedicated }} cluster to v23.1.11, please [submit a support request](https://support.cockroachlabs.com/hc/en-us/requests/new?).
```

Reply:
This is a nit/preference, feel free to ignore. Because, we outline that it is **not** available for self-hosted, the "only" doesn't seem necessary.